scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1334,6 +1365,7 @@
                         ;; TODO: Rescind the offer in fenzo
                         )
        (framework-message [this driver executor-id slave-id message]
+                          (future (handle-framework-message conn message))
```

Comment:
How important is `future` here?  Is it not important to ensure framework messages from about the same task a processed in order?